datadog_checks_base/datadog_checks/base/utils/db/query.py

```diff
@@ -21,17 +23,22 @@ class Query(object):
 
     def __init__(self, query_data):
         # type: (Dict[str, Any]) -> Query
+        # Contains the data to fill the rest of the attributes
         self.query_data = deepcopy(query_data or {})  # type: Dict[str, Any]
         self.name = None  # type: str
+        # The actual query
         self.query = None  # type: str
-        self.columns = None  # type: List[str]
-        self.extras = None  # type: List[Dict[str, str]]
-        self.tags = None  # type: List[str]
+        # Contains a mapping of column_name -> column_type, transformer
+        self.column_transformers = None  # type: Tuple[Tuple[str, Tuple[str, Transformer]]]
+        # These transformers are used to collect extra metrics calculated from the query result
+        self.extra_transformers = None  # type: List[Tuple[str, Transformer]]
+        # Contains the tags defined in query_data not the ones returned by the query
+        self.custom_tags = None  # type: List[str]
```

Comment:
This isn't always just what the user defines so I think `tags` is better/less confusing

Reply:
Where else can they come from?